source/StopWatch/UI/WorklogForm.cs

```diff
@@ -89,21 +89,18 @@ public int RemainingEstimateSeconds
 
 
         #region public methods
-        public WorklogForm(DateTimeOffset startTime, TimeSpan TimeElapsed, string comment, EstimateUpdateMethods estimateUpdateMethod, string estimateUpdateValue)
-        {            
-            this.TimeElapsed = TimeElapsed;
-            DateTimeOffset initialStartTime;
-            if (startTime == null)
+        public WorklogForm(IssueControl issueControl)
```

Comment:
I don't think it's good practice to increase coupling.

Reply:
I did this to not make the already long signature of the constructor even longer. Because all the arguments only really make sense if they stem from the same IssueControl object I thought the cleaner signature makes clearer what's going on.

It's not a big deal to change that back, though – if this PR has any chance of being accepted by @carstengehling.

Reply:
Agree with you. Repository looks abandoned...